### PR TITLE
Fix "To-Many" relationship example snippet

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- reported through feedback widget

### Staged Changes (Requires MongoDB Corp SSO)

- [Relationships - Kotlin SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/to-many-relationship-fix/sdk/kotlin/realm-database/schemas/relationships/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by mongodben — APPROVED** [review]
lgtm

**Comment by nlarew** [comment]
Any idea why the tests are falling? Otherwise LGTM

File: examples/kotlin/shared/src/commonTest/kotlin/com/mongodb/realm/realmkmmapp/SchemaTest.kt [hunk 1]
```diff
@@ -40,7 +41,7 @@ class Fish: RealmObject {
 class Sushi: RealmObject {
     val _id: ObjectId = ObjectId.create()
     val name: String = ""
-    val fishes: List<Fish> = listOf()
+    val fishes: RealmList<Fish> = realmListOf()
```
Comment (nathan-contino-mongo) [inline examples/kotlin/shared/src/commonTest/kotlin/com/mongodb/realm/realmkmmapp/SchemaTest.kt:44]:
```suggestion
    val fishes: RealmList<Fish> = realmListOf<Fish>()
```

File: source/examples/generated/kotlin/SchemaTest.snippet.to-many-relationship.kt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class Sushi: RealmObject {
     val _id: ObjectId = ObjectId.create()
     val name: String = ""
-    val fishes: List<Fish> = listOf()
+    val fishes: RealmList<Fish> = realmListOf()
```
Comment (nathan-contino-mongo) [inline source/examples/generated/kotlin/SchemaTest.snippet.to-many-relationship.kt:4]:
```suggestion
    val fishes: RealmList<Fish> = realmListOf<Fish>()
```

**Pull request merged**